src/lib/translators/mongo.ts

```diff
@@ -222,6 +223,43 @@ function transformTop(step: TopStep): Array<MongoStep> {
   ];
 }
 
+/** transform an 'unpivot' step into corresponding mongo steps */
+function transformUnpivot(step: UnpivotStep): Array<MongoStep> {
+  let mongoPipeline: Array<MongoStep> = [];
+  let projectCols: PropMap<string> = {};
+  let objectToArray: PropMap<string> = {};
```

Comment:
I think we could use `const` here:

```suggestion
  const objectToArray: PropMap<string> = {};
```